mimxrt685_evk_cm33: Build failed on tests/drivers/spi/spi_loopback/drivers.spi.loopback

**Describe the bug**
Build in CI fails due to following error:

```
zephyr/drivers/dma/CMakeFiles/drivers__dma.dir/dma_mcux_lpc.c.obj -c /__w/zephyr/zephyr/drivers/dma/dma_mcux_lpc.c
/__w/zephyr/zephyr/drivers/dma/dma_mcux_lpc.c: In function 'dma_mcux_lpc_reload':
/__w/zephyr/zephyr/drivers/dma/dma_mcux_lpc.c:383:29: error: 'data' undeclared (first use in this function)
  383 |  uint32_t virtual_channel = data->channel_index[channel];
      |                             ^~~~
/__w/zephyr/zephyr/drivers/dma/dma_mcux_lpc.c:383:29: note: each undeclared identifier is reported only once for each function it appears in
/__w/zephyr/zephyr/drivers/dma/dma_mcux_lpc.c:382:32: error: unused variable 'dev_data' [-Werror=unused-variable]
  382 |  struct dma_mcux_lpc_dma_data *dev_data = dev->data;
      |                                ^~~~~~~~
cc1: all warnings being treated as errors
```

**Impact**
Blocks CI, cf https://github.com/zephyrproject-rtos/zephyr/runs/4934340162?check_suite_focus=true

--- a/drivers/dma/dma_mcux_lpc.c
+++ b/drivers/dma/dma_mcux_lpc.c
@@ -380,7 +380,7 @@ static int dma_mcux_lpc_reload(const struct device *dev, uint32_t channel,
 			       uint32_t src, uint32_t dst, size_t size)
 {
 	struct dma_mcux_lpc_dma_data *dev_data = dev->data;
-	uint32_t virtual_channel = data->channel_index[channel];
+	uint32_t virtual_channel = dev_data->channel_index[channel];
 	struct call_back *data = DEV_CHANNEL_DATA(dev, virtual_channel);
 	uint8_t src_inc, dst_inc;
 	uint32_t xferConfig = 0U;

